lagunita/lms/templates/google_analytics.html

```diff
@@ -4,7 +4,6 @@
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-  ga('create', 'UA-40837107-1', 'class.stanford.edu');
-  ga('send', 'pageview');
-
+  ga('create', 'UA-40837107-1', 'class.stanford.edu', {'allowAnchor': true});
+  ga('send', 'pageview', { 'page': location.pathname + location.search + location.hash});
```

Comment:
let's change this to: `ga('send', 'pageview', (location.pathname + location.search + location.hash));`